packages/web3-react/src/hooks/useWallet.tsx

```diff
@@ -20,22 +20,20 @@ import { useAlephiumConnectContext } from '../contexts/alephiumConnect'
 import { NodeProvider, SignerProvider, Account, NetworkId } from '@alephium/web3'
 
 export interface Wallet {
-  signer: SignerProvider
-  account: Account & { network: NetworkId }
-  nodeProvider?: NodeProvider
+  signer: SignerProvider | undefined
+  account: (Account & { network: NetworkId }) | undefined
+  nodeProvider: NodeProvider | undefined
 }
 
 export function useWallet() {
-  const { account, signerProvider } = useAlephiumConnectContext()
+  const { account, signerProvider, network } = useAlephiumConnectContext()
```

Comment:
When auto reconnecting, it is possible that the `account` is not undefined but the `signerProvider` is undefined, so I changed the return value of the `useWallet` hook.